docs/files.md

```diff
@@ -297,6 +297,14 @@ Location: http://cozy.example.com/files/9152d568-7e7c-11e6-a377-37cbfb190b4b
           "type": "io.cozy.files",
           "id": "fce1a6c0-dfc5-11e5-8d1a-1f854d4aaf81"
         }
+      },
+      "references": {
```

Comment:
[naming] `referenced_by` ? `references` would be the invert relationship, from doc to files.